src/Raven.Studio/typescript/commands/database/settings/getSchemaValidationCommand.ts

```diff
@@ -0,0 +1,17 @@
+import commandBase = require("commands/commandBase");
+import database = require("models/resources/database");
+import endpoints = require("endpoints");
+class getSchemaValidationCommand extends commandBase {
+    constructor(private db: database | string) {
+        super();
+    }
+
+    execute(): JQueryPromise<Raven.Client.Documents.Operations.SchemaValidation.SchemaValidationConfiguration> {
+        const url = endpoints.databases.schemaValidation.schemaValidationConfig;
+
+        return this.query<Raven.Client.Documents.Operations.SchemaValidation.SchemaValidationConfiguration>(url, null, this.db)
+            .fail((response: JQueryXHR) => this.reportError("Failed to get schema validations", response.responseText, response.statusText));
```

Comment:
When there is no configuration (clean db) server returns 404. It should be handled here

<img width="371" height="193" alt="Image" src="https://github.com/user-attachments/assets/a122a77c-0ecb-4554-a2d7-c0b4770728b4" />